src/data/auth/authServices.ts

```diff
@@ -1,3 +1,52 @@
+import {
+  SignUpUserData,
+  SignUnUserResponse,
+  SignInUserData,
+  SignInUserResponse,
+  RefreshTokenData,
+  RefreshTokenResponse,
+  LogoutUserResponse,
+} from '../../types/auth/auth';
+import axiosInstance from '../http/http';
 import { signInUserMock } from './__mocks/authServicesMocks';
 
 export const signInUser = signInUserMock;
+
+export const signUpUser = (
+  signUpUserData: SignUpUserData,
+): Promise<SignUnUserResponse> => {
+  return axiosInstance
+    .post<SignUnUserResponse>(`signup`, signUpUserData)
+    .then((res) => {
+      return res.data;
```

Comment:
Please  create a ticket is Jira for improvements: Create and http methods which will returns  response.data. It will be low priority ticket.
It will allow to do like this:

```typescript
export const signUpUser = (
  signUpUserData: SignUpUserData,
): Promise<SignUnUserResponse> => {
  return await post<SignUnUserResponse>(`signup`, signUpUserData);
```